db/migrate/20170207215322_fix_vpor_time_profile_ids.rb

```diff
@@ -0,0 +1,44 @@
+class FixVporTimeProfileIds < ActiveRecord::Migration[5.0]
+  class VimPerformanceOperatingRange < ActiveRecord::Base
+  end
+
+  class TimeProfile < ActiveRecord::Base
+    ALL_DAYS   = (0...7).to_a.freeze
+    ALL_HOURS  = (0...24).to_a.freeze
+    DEFAULT_TZ = "UTC".freeze
+
+    serialize :profile, Hash
+
+    def self.default
+      @default ||= begin
+        ar_region_class = ArRegion.anonymous_class_with_ar_region
+        region_cond = ar_region_class.region_to_conditions(ar_region_class.my_region_number)
+
+        where(region_cond)
+          .where(:rollup_daily_metrics => true)
+          .select do |tp|
+            tp.profile[:days].try(:sort) == ALL_DAYS &&
+              tp.profile[:hours].try(:sort) == ALL_HOURS &&
+              tp.profile[:tz] == DEFAULT_TZ
+          end
```

Comment:
Do we also need `tp.profile_type == "global" here to ensure it's the default one that we match?